src/Juvix/Compiler/Core/Normalizer.hs

```diff
@@ -0,0 +1,177 @@
+module Juvix.Compiler.Core.Normalizer where
```

Comment:
What do you think if we better place this file under the transformation directory?

Reply:
We can, depends how you look at it. I think of it as more analogous to evaluation than an intermediate transformation on Core.

Reply:
Okey.